src/org/usfirst/frc/team694/robot/commands/ToggleBITCOINAutomationCommand.java

```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+
+/**
+ *
+ */
+public class ToggleBITCOINAutomationCommand extends InstantCommand {
```

Comment:
This is all cool and good, but based off of the Operator Pad layout, there will not be a toggle for turning BITCOIN on and off, but rather two individual buttons, one for turning it off, and one for turning it on. You don't have to delete this command, but make two more to reflect that layout.